content/en/blog/2023/announcing-major-release.md

```diff
@@ -0,0 +1,106 @@
+- Sign artifacts as well as list and inspect signatures stored in OCI-compliant registries
+    - Compliant with [`image-spec v1.0.2`](https://github.com/opencontainers/image-spec/tree/v1.0.2)
+    - Compliant with [`distribution-spec v1.0.1`](https://github.com/opencontainers/distribution-spec/tree/v1.0.1)
+    - Compatible with [`image-spec v1.1.0-rc4`](https://github.com/opencontainers/image-spec/tree/v1.1.0-rc4)
+    - Compatible with [`distribution-spec v1.1.0-rc3`](https://github.com/opencontainers/distribution-spec/tree/v1.1.0-rc3) (limited to [referrers tag schema](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc3/spec.md#referrers-tag-schema))
+- Support two signature envelope formats 
+    - [COSE](https://github.com/notaryproject/notaryproject/blob/v1.0.0/specs/signature-envelope-cose.md): it provides message security assurance for services like IoT applications and using [CBOR](https://datatracker.ietf.org/doc/html/rfc8152) as the message-encoding format
+    - [JWS](https://github.com/notaryproject/notaryproject/blob/v1.0.0/specs/signature-envelope-jws.md): JWS is a JSON based envelope format for digital signatures that can be used when required for interoperability (mostly OIDC)
+
+From the software consumer's perspective, verifying the signature of a signed artifact ensures its integrity and authenticity. Notary Project provides the following core capabilities for verification experience:
+
+- Verify signatures using [trust store and trust policy](https://github.com/notaryproject/specifications/blob/v1.0.0/specs/trust-store-trust-policy.md) with fine-tuned OCI repository specific trust policies, and support for various enforcement levels for signature verification (e.g. `enforce`, `permissive`, `audit`) to enable a wide range of scenarios. 
+- [notation policy](https://notaryproject.dev/docs/cli-reference/notation_policy/) command is also introduced in this release to simplify the experience of importing and inspecting the trust policy.
+
+### Experimental features
+
+Experimental features are intended for testing and evaluation purposes only and should not be used in production environments. Users can enable experimental features in Notation CLI by setting the environment variable `NOTATION_EXPERIMENTAL` to 1 as shown below.
+
+```
+export NOTATION_EXPERIMENTAL=1
+```
+
+There are two major features which are marked as experimental but might be useful in some specific scenarios.
+ 
+- [Signing, listing and verification with OCI image layout](https://notaryproject.dev/docs/how-to/oci-image-layout/) for signing images before they are pushed to a registry. It enables users sign and verify artifacts stored in the local file system.
+- [OCI distribution referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc2/spec.md#enabling-the-referrers-api), which allows Notation CLI to fetch a list of signatures in an efficient and clean manner.
```

Comment:
I would suggest removing ", which" from the sentence to make it more readable.

Reply:
```suggestion
- [OCI distribution referrers API](https://github.com/opencontainers/distribution-spec/blob/v1.1.0-rc2/spec.md#enabling-the-referrers-api). This allows the Notation CLI to fetch a list of signatures in an efficient and clean manner.
```